.github/workflows/locker.yml

```diff
@@ -0,0 +1,36 @@
+name: Locker - Lock stale issues and PRs
+on:
+  schedule:
+    - cron: '0 9 * * *' # Once per day, early morning PT
+
+  workflow_dispatch:
+    # Manual triggering through the GitHub UI, API, or CLI
+    inputs:
+      daysSinceClose:
+        required: true
+        default: "30"
+      daysSinceUpdate:
+        required: true
+        default: "30"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v4
```

Comment:
We use `checkout@v3`:
https://github.com/dotnet/extensions/blob/13a6288c9de8d8f83c40ae419803e6a990bf5825/.github/workflows/azure-sync.yml#L19

Can we use the same? Or we specifically need v4?

We can probably update the others to v4, if required...

Reply:
I do not know if v4 was _required_ here. It's a straightforward git clone, so I suspect it would work. Since this PR has already been merged, I'll leave it to you to update this or other actions as you see fit though, @RussKie.